src/codestyle.sh

```diff
@@ -73,6 +73,49 @@ function codestyle_main()
   checkpatch=$(join_path "$kernel_root" 'scripts/checkpatch.pl')
   cmd_script="perl ${checkpatch} ${checkpatch_options}"
 
+  [[ -n "${options_values['START_LINE']}" ]] && start_line=${options_values['START_LINE']}
+  [[ -n "${options_values['END_LINE']}" ]] && end_line=${options_values['END_LINE']}
+
+  if [[ -n "$start_line" || -n "$end_line" ]]; then
+    range_option_flag='true'
```

Comment:
You should declare this variable as local and also initialize it with an empty string.